bazel/utils/testdata/dynamic/hello-dynamic.cc

```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+#include <gtest/gtest.h>
+
+int main(int argc, char** argv) {
+  // Ensure a function from gtest is included.
+  // Using argc so the compiler cannot optimize it out.
+  EXPECT_EQ(1, argc);
+
+  std::cout << "Hello, world!" << std::endl;
+}
```

Comment:
a `return 0;` here just to be safe?

Reply:
good catch, done!

Reply:
well, this has been changed to check for argv and env, so now returns the correct status.